### PR TITLE
Kalia Network gentx & peer

### PR DESCRIPTION
**Pull request by KaliaNetwork**

### CODE REVIEW

File: quark/gentxs/gentx-KaliaNetwork.json [hunk 1]
```diff
@@ -0,0 +1 @@
...oniker":"Kalia Network","identity":"3D6E2861B47F2F9F","website":"https://kalia.network","security_contact":"info@kalia.network","details":"Kalia Network validator node. Delegate your tokens and Start Earning Staking Rewards"},"commission":{"rate":"0.050000000000000000","max_rate":"1.000000000000000000","max_change_rate":"1.000000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1zd6z7r658lyck4qgp0vasjhl7e2whnt3gevvla","validator_address":"neutronvaloper1zd6z7r658lyck4qgp0vasjhl7e2whnt3jy44ee","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"yQ0g4YEn0hSFZV7B9YU+hFcxGlwmLPyIRT+dbsCxxNI="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"c8b0794e4a7da67f4c98f0115adbbfd0fc89dfb4@46.4.81.204:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A4ay4Fppb0D85aUIgF/bdNpFVmIPqvlYAk2afDmviSgO"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["gYPOwtkbKj2Z5eXwxB1nCjUM9lPHi0zydDGlLfCoK5BxHl0UJ3YjcFU+4kISYC0x7P+LFymQYgG4g5ecZc9UfQ=="]}
```
Comment (albertandrejev) [inline quark/gentxs/gentx-KaliaNetwork.json:1]: Please update `max_change_rate` and `max_rate` in accordance with the documentation.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/gentx-KaliaNetwork.json (hunk 1 above)
Comment (KaliaNetwork) [reply quark/gentxs/gentx-KaliaNetwork.json:1]: Dear albert, we have opened a ticket in Discord. Could you check, please?

Comment (KaliaNetwork) [reply quark/gentxs/gentx-KaliaNetwork.json]: gentx is updated

**Comment by KaliaNetwork** [comment]
- Kalia Network
- https://kalia.network/
- Kalia Network#1013   and   sr#3753
- neutronvaloper1zd6z7r658lyck4qgp0vasjhl7e2whnt3jy44ee

**Review by albertandrejev — APPROVED** [review]

**Pull request merged**